crengine/src/lvtinydom.cpp

```diff
@@ -7774,6 +7775,8 @@ void ldomNode::initNodeRendMethod()
             }
         }
     #endif
+
+    persist();
```

Comment:
Asking, as you must have dig into that to understand it - and you added more in this and previous commit:
Is there a simple rule of thumb as to when we should use/add it?
No slowness added with doing it for each node here?

Reply:
> Asking, as you must have dig into that to understand it - and you added more in this and previous commit:
> Is there a simple rule of thumb as to when we should use/add it?

I think yes. If you call `ldomNode::modify ()` to perform any node manipulation, then after that you must call `ldomNode::persist()`.

> No slowness added with doing it for each node here?

I don't know, not tested.

Reply:
Just look at the `ldomNode::setAttributeValue()` function where `ldomNode::modify()` is called. I think it would be silly to call `persist()` right there, it can be done later.

Reply:
See `ldomDocumentWriter::ElementCloseHandler()`.

Also you can try `tinyNodeCollection::persist()` instead of this.